corehq/apps/cloudcare/templates/formplayer/case_list.html

```diff
@@ -19,8 +19,11 @@
     </div>
     <% } %>
 
-    <div style="<%- columnStyle %>">
-      <div id="module-case-list-column-next-to-map">
+    <div class="row row-no-gutters">
+      <% if (showMap) { %>
+        <div id="module-case-list-map" class="col-sm-5 col-sm-push-9<% if (useTiles) { %> white-border<% } %>"></div>
```

Comment:
Is the `push` and `pull` going to be deprecated in bootstrap5? If so, I know there's the transition happening, might want to reach out to Biyeun to see how to handle it

Reply:
100% it's deprecated, and I'm glad for it. In bootstrap 5 these can be replaced by `order-sm-1` and `order-sm-2` to specify the order they should be displayed in at a given breakpoint. Good call on reaching out to Biyeun, will do that

Reply:
Per Biyeun, it'll be some time until changes start to come to web apps, so no need to account for Bootstrap 5 here yet.